client/src/components/Onboarding/OnboardingActions.scss

```diff
@@ -1,5 +1,9 @@
 .onboarding-action__loading {
   display: inline-block;
-  width: inherit;
+  width: auto;
```

Comment:
Short version: The width: auto is a simpler, and equally reliable (here), solution that gives the same result.

Long version: https://github.com/TACC/Core-Portal/pull/370#discussion_r599110936